ams/api/src/main/java/com/netease/arctic/ams/api/process/AmoroProcess.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.ams.api.process;
+
+import com.netease.arctic.ams.api.Action;
+
+/**
+ * AmoroProcess is a process the whole lifecycle of which is managed by Amoro. The process is
+ * submitted by user or system and handled by Amoro. The process could be a minor optimizing, a
+ * major optimizing, an external optimizing, a snapshot refresh, a snapshot expire, a clean orphaned
+ * files or a hive commit sync.
```

Comment:
Is it possible to add doc `{@link Action}`?

Reply:
Have improved this comment